### DISCUSSION TITLE
Can I use the same session layer for different axum servers?

### DISCUSSION THREAD

**Discussion by EMUNES** [maxcountryman/tower-sessions#199] [Q&A]
I started two axum servers, with the same session layer (wrapped in oncelock), and I inserted a cookie in one server. When I try to extract the session information from another server, it seems to fail. I can only validate the session in the same server where I insert it.

Is this normal, or there are things for me to work if I want to make them right?

**Comment by maxcountryman** [comment]
Please provide a minimal example. You can use one of the examples in this repo: modify it to demonstrate what you're trying to do and share it so we can take a look.
